src/main/resources/lang/polish.lang

```diff
@@ -0,0 +1,189 @@
+			io error: Błąd podczas włączania skryptów w folderze <gold>%s<red> (niektóre skrypty mogą zostać włączone po restarcie serwera): %s
+	disable:
+		all:
+			disabled: Pomyślnie wyłączono wszystkie skrypty.
+			io error: Nie udało się nazwać wszystkich skryptów ponownie - niektóre skrypty mogą zostać włączone po restarcie serwera: %s
+		single:
+			already disabled: <gold>%s<reset> jest już wyłączony!
+			disabled: Pomyślnie wyłączono <gold>%s<reset>.
+			io error: Nie udało się nazwać <gold>%s<red> ponownie, skrypt może zostać włączony po restarcie serwera: %s
+		folder:
+			empty: <gold>%s<reset> nie zawiera żadnych włączonych skryptów.
+			disabled: Pomyślnie wyłączono <gold>%2$s<reset> skrypt¦¦ów¦ w <gold>%1$s<reset>.
+			io error: Nie udało się wyłączyć wszystkich skryptów w <gold>%s<red> (niektóre skrypty mogą zostać wyłączone po restarcie serwera): %s
+	update:
+		# check/download: see Updater
+		changes:
+			# multiple versions:
+			# 	title: <gold>%s<r> aktualizacj¦a¦i¦ zostało wydanych od wydania tej wersji (<gold>%s<r>) Skript'a:
+			# 	footer: Aby pokazać listę zmian wersji, napisz <gold>/skript update changes <version><reset>
+			# invalid version: Nie ma listy zmian dla wersji <gold>%s<red>
+			title: <bold><cyan>%s<reset> (%s)
+			next page: <grey>strona %s. z %s. Wpisz <gold>/skript update changes %s<gray> by przewinąć stronę (podpowiedź: użyj strzałki w górę na klawiaturze)
+	info:
+		aliases: Aliasy Skript'a mogą zostać znalezione tutaj: <aqua>https://github.com/SkriptLang/skript-aliases
+		documentation: Dokumentacja Skript'a może zostać znaleziona tutaj: <aqua>https://skriptlang.github.io/Skript
+		version: Wersja Skript: <aqua>%s
+		server: Wersja serwera: <aqua>%s
+		addons: Zainstalowane dodatki Skript: <aqua>%s
+		dependencies: Zainstalowane powiązane wtyczki: <aqua>%s
+
+# -- Updater --
+updater:
+	not started: Skript jeszcze nie sprawdził najnowszej stabilnej wersji. Użyj <gold>/skript update check<reset> aby to zrobić.
+	checking: Sprawdzanie najnowszej wersji Skript'a...
+	check in progress: Sprawdzanie najnowszej wersji Skript'a obecnie trwa.
+	updater disabled: Aktualizator jest obecnie wyłączony, więc sprawdzenie najnowszej wersji Skript'a nie powiodło się.
+	check error: <red>Znaleziono błąd podczas sprawdzania najnowszej wersji Skript'a:<light red> %s
+	running latest version: Na twoim serwerze jest obecnie najnowsza stabilna wersja Skript'a.
+	running latest version (beta): Na twoim serwerze obecnie jest wersja <i>beta<r> Skript'a, i jeszcze nie ma dostępnej <i>stabilnej<r> wersji. Do nowszych wersji beta musisz aktualizować plugin ręcznie!
+	update available: Nowa wersja Skript'a dostępna: <gold>%s<reset> (obecnie na twoim serwerze jest <gold>%s<reset>)
+	downloading: Pobieranie Skript <gold>%s<reset>...
+	download in progress: Najnowsza wersja Skript'a jest obecnie pobierana.
+	download error: <red>Znaleziono błąd podczas pobierania najnowszej wersji Skript'a:<light red> %s
```

Comment:
```suggestion
	download error: <red>Wystąpił błąd podczas pobierania najnowszej wersji Skript'a:<light red> %s
```
Znaleziono błąd to intrygujące określenie :) *Szuka, szuka błędów... o, w końcu znalazłem!*